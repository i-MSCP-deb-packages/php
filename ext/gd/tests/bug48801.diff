001+ (1, 15)
002+ (162, 15)
003+ (162, -48)
004+ (1, -48)
001- (-1, 15)
002- (155, 15)
003- (155, -48)
004- (-1, -48)