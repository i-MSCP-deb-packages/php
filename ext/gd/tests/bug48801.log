
---- EXPECTED OUTPUT
(-1, 15)
(155, 15)
(155, -48)
(-1, -48)
---- ACTUAL OUTPUT
(1, 15)
(162, 15)
(162, -48)
(1, -48)
---- FAILED
