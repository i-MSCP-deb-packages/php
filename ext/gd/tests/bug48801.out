(1, 15)
(162, 15)
(162, -48)
(1, -48)